_posts/2015-10-08-unacceptable-behaviors.md

```diff
@@ -15,3 +15,5 @@ The same goes for [tone policing](http://tooyoungforthelivingdead.tumblr.com/ton
 The language we use every day is frequently tied to a past of violence and imperialism; many of these assumptions are built into the technology we use (the internet itself began as a [US military effort](https://en.wikipedia.org/wiki/DARPA)). Without thinking about their origins, we may use military ranks to refer to people, refer to a crisis evaluation as a “war room,” talk about a shocking event as a “bombshell,” and so on. In consideration of your colleagues’ life experiences and how they may be affected by such language, commit to finding neutral replacements for these words in your work, and [finding ways to limit exposure to them](https://medium.com/@mikebroberts/let-s-stop-saying-master-slave-10f1d1bf34df) via third-party sources. When reviewing these sources, review the terminology they use with the same care you would review their technical specifications.
 
 There are a host of behaviors and language common on tech teams which are worth noting as specifically unwelcome: Avoid “_well, actuallys_”—pedantic corrections that are often insulting and unproductive; make an effort not to interrupt your colleagues while they are speaking; never respond with surprise when someone asks for help; and take care neither to patronize your colleagues nor assume complete knowledge of a topic. This last point is especially important when talking about technical topics: Many women and people of color in the tech industry have many tales of being either [mansplained](https://www.guernicamag.com/daily/rebecca-solnit-men-explain-things-to-me/) about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions—don’t be that person. Remember that your colleagues may have expertise you are unaware of, and listen at least as much as you speak.
+
+There will be cases where you may will work with customers, service providers or other third parties who may not hold themselves to the standard outlined here. While there is no expectation that you should accept transgressive behavior simply because it comes from outside your team, take care to not reinforce adversarial or toxic relationships by engaging in excessive complaining or disrespectful comments in discussions. Acknowledge, report and attempt to correct behavior, but be careful not to let your response become counter-productive or destructive to morale.
```

Comment:
```suggestion
There will be cases where you may work with customers, service providers or other third parties who may not hold themselves to the standard outlined here. While there is no expectation that you should accept transgressive behavior simply because it comes from outside your team, take care to not reinforce adversarial or toxic relationships by engaging in excessive complaining or disrespectful comments in discussions. Acknowledge, report and attempt to correct behavior, but be careful not to let your response become counter-productive or destructive to morale.
```

> Acknowledge, report and attempt to correct behavior

Those are valid options, but I don't think the intention is to say that we are required to take these actions, is it?